packages/compiler/lib/std/visibility.tsp

```diff
@@ -141,15 +141,22 @@ extern dec withVisibility(target: Model, ...visibilities: valueof (string | Enum
 extern dec withDefaultKeyVisibility(target: Model, visibility: valueof string | EnumMember);
 
 /**
- * Sets which visibilities apply to parameters for the given operation.
+ * Declares the visibility constraint of the parameters of a given operation.
  *
- * @param visibilities List of visibility strings which apply to this operation.
+ * A parameter or property nested within a parameter will be visible if it has _any_ of the visibilities
+ * in the list, or if the list is empty (in which case the parameter or property is always visible).
```

Comment:
Were you going to describe the weird behavior of an empty list here?

Reply:
I added a blurb to this that communicates the behavior and its strangeness, and cautions against using it.